README.md

```diff
@@ -1,10 +1,59 @@
-# Frequenz repository common configuration for Python
+# Frequenz Project Setup Tools
 
-This is very opinionated set of tools and configurations to setup a Python
-repository for Frequenz projects.
+Frequenz project setup tools is a collection of tools and common configurations for setting up and managing different types of repositories in the Frequenz ecosystem. These tools are particularly designed to handle 4 main types of repositories:
 
-If offers:
+- APIs (api)
+- Actors (actor)
+- Applications (app)
+- Libraries (lib)
 
-* Trivial build of `noxfile.py` with some predefined sessions with all common
-  checks.
-* Tools to build protobuf/grpc files as Python, including type information.
+## Installation
+
+The Frequenz project setup tools can be installed via pip:
+
+~~~bash
```

Comment:
Just curious, why `~~~`? I didn't even knew it was a supported syntax.

Reply:
Sometimes you forget to press shift,  then you get ~~~

Reply:
Mostly because that way chatGPT wouldn't render it but give it in a nice copy-able block :grin: